src/channel.h

```diff
@@ -83,7 +83,7 @@ class CChannel : public QObject
     void SetEnable ( const bool bNEnStat );
     bool IsEnabled() { return bIsEnabled; }
 
-    void                SetAddress ( const CHostAddress NAddr ) { InetAddr = NAddr; }
+    void                SetAddress ( const CHostAddress& NAddr ) { InetAddr = NAddr; }
```

Comment:
Is this one OK?  If the caller changes the content of the reference, the value on the channel would change.  Is that safe or should this actually be a "saved copy"?

Reply:
This is fine, it's not like a pointer. The reference is dereferenced as part of fetching `NAddr`, and it is the dereferenced value that is stored in `InetAddr`.

Reply:
Ah right.